scipy/stats/_continuous_distns.py

```diff
@@ -6655,7 +6655,7 @@ def _preprocess(self, x, skew):
         # close to this small.
         norm2pearson_transition = 0.000016
 
-        ans, x, skew = np.broadcast_arrays([1.0], x, skew)
+        ans, x, skew = np.broadcast_arrays(1.0, x, skew)
```

Comment:
What does this have to do with scalar conversion? IIUC, none of the three variables are being coerced from array to float, right?

Reply:
Not at this point, but using `[1.0]` instead of `1.0` adds an unnecessary dimension ifi `x` and `skew` are scalars. This then leads to issues down the line.

Reply:
Ah, OK. Makes sense.